src/runtime/module_jit_ref_count.cpp

```diff
@@ -0,0 +1,29 @@
+#include "./runtime_internal.h"
+
+extern "C" {
+
+WEAK void *halide_jit_module_argument = NULL;
+WEAK void (*halide_jit_module_adjust_ref_count)(void *arg, int32_t count) = NULL;
+
+}
+
+namespace Halide { namespace Runtime { namespace Internal {
+
+WEAK void halide_use_jit_module() {
```

Comment:
(Also, as Andrew pointed out, the code here should be ok due to Buffer holding a reference on the module. I think this PR can be merged independently of gpu_api_naming.)